examples/toolkit_showcase/toolkit_showcase.ipynb

```diff
@@ -24,7 +24,7 @@
     {
```

Comment:
<p>(blocking) Could you confirm that this warning is intentional? This is a somewhat strange thing to see given that we haven't used mbuild or committed to exporting to LAMMPS. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openforcefield/openff-toolkit/pull/1981/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>